manifests/i/iMobieInc/Ashampoo/26.00.22/iMobieInc.Ashampoo.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.2.13 $debug=AUSU.CRLF.5-1-19041-3803.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: iMobieInc.Ashampoo
+PackageVersion: 26.00.22
+InstallerType: inno
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cdn1.ashampoo.net/public/imo/6706/ashampoo_winoptimizer_26_38898.exe
+  # InstallerUrl: https://www.imobie.com/go/download.php?product=ashampoo&id=0
+  InstallerSha256: 0568524591C97EB3D38D84CD711D98D7CC0F8A2992ECAF6B15A7EB8B6FAF8D4D
```

Comment:
```suggestion
  InstallerSha256: CC65F50E4DC257A21BEAFD4CB7BEA1AEBF201A7037436812ABED61A8AA274368
```

Reply:
@stephengillie - Can you check the hash you get when you download it to your VM? I always get 8D4D

Reply:
```PowerShell
PS C:\ManVal> get-fileHash '.\Installers\ashampoo_winoptimizer_26_38898 (1).exe'

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          0568524591C97EB3D38D84CD711D98D7CC0F8A2992ECAF6B15A7EB8B6FAF8D4D       C:\ManVal\
```

Reply:
Unfortunately, the hash has to match at the pipelines, or critical scans can't be performed.